features/schema.py

```diff
@@ -142,7 +142,7 @@ class Meta:
     def filter_updated_since(self, queryset, name, value):
         return queryset.filter(
             Q(overrides__modified_at__gt=value) | Q(source_modified_at__gt=value)
-        )
+        ).distinct()
```

Comment:
Maybe consider commenting the code? The need for distinct isn't immediately obvious. If overrides get refactored later to having a single timestamp (for example), then it would be more clear distinct could be removed here.

Reply:
A fair point. I'll add some comments.

Reply:
Added some comments.